store/storetest/thread_store.go

```diff
@@ -693,7 +592,7 @@ func testThreadStorePermanentDeleteBatchThreadMembershipsForRetentionPolicies(t
 	// Delete team policy and thread
 	err = ss.RetentionPolicy().Delete(teamPolicy.ID)
 	require.NoError(t, err)
-	err = ss.Thread().Delete(post.Id)
+	_, err = s.GetMaster().Exec("DELETE FROM Threads WHERE PostId='" + post.Id + "'")
```

Comment:
One quick comment - let's please use `GetMasterX()` instead. :) We are slowly moving away from gorp.

Reply:
Awesome to see us moving away from gorp! One challenge here is that the `SqlStore` interface used in the tests only defines `GetMaster`, and it looks like `GetMasterX` returns a `sqlxDBWrapper` not presently exported by the `sqlstore` package. Should we tackle this holistically in the tests?

Reply:
Oh yeah, this PR https://github.com/mattermost/mattermost-server/pull/19545 hasn't gone in yet. I'll let you know if that goes in first. Otherwise, we can keep this be for now.

Reply:
Done!